non_iid_detection/non_iid_datalab.ipynb

```diff
@@ -0,0 +1,1248 @@
+{
```

Comment:
when showing images, show more than 4 images per column, and maybe only 2 rows of them. Clarify in text they're ordered left-right in the dataset